actions/playerHeroesActions.js

```diff
@@ -1,8 +1,10 @@
 import fetch from 'isomorphic-fetch';
 import { HOST_URL } from './';
 import { playerHeroes } from '../reducers';
+import { getUrl, addQueryString, defaultOptions, getModifiedOptions } from './utility';
 
-const url = (playerId, queryString) => `/api/players/${playerId}/heroes?${queryString}`;
+const excludedOptions = ['limit'];
```

Comment:
whenever I pass back the limit query param to heroes, it limits the data on heroes. Since heroes is an aggregation I assumed that we might not want that behavior. I might not want all my matches back on overview, for example, but I want all the hero data (if that makes sense). I'm not sure what the usecase would be for limiting the hero data except if we were able to sort serverside the matches returned (top 100 games sorted by kills or something).